src/utils.ts

```diff
@@ -89,15 +89,11 @@ export function getDatabaseViewDataUrl(
   pathname: string
 ): string {
   const urlObj = new url.URL(origin);
-  if (
-    urlObj.hostname.includes("firebaseio.com") ||
-    urlObj.hostname.includes("firebasedatabase.app")
-  ) {
+  if (urlObj.hostname.includes("firebase")) {
```

Comment:
So that `FIREBASE_REALTIME_URL=https://firebaseio-staging.com` will work.

Reply:
Let's just list all three full domain names to be on the safe side. `myfirebaseapp.example.com` should be filtered out.

Reply:
Is there a way to detect emulator specific url instead? It might be cleaner.
Are they always "localhost"?

Reply:
No, in theory the emulator can listen on all sorts of hosts and ports. Common configurations include `127.0.0.1`, `localhost`, but it can be changed by the user to LAN or even public IP addresses and domains, and I've seen those in the wild. We had a similar discussion re: RTDB emulator and the conclusion so far is not to try and detect emulator URLs

Reply:
SG. I will just check for `firebaseio` or `firebasedatabase` if u think `firebase` is too general.